src/main/java/cat/udl/eps/softarch/academicrecruit/domain/Participant.java

```diff
@@ -0,0 +1,41 @@
+package cat.udl.eps.softarch.academicrecruit.domain;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class Participant extends UriEntity<Long> {
+
+    public enum Role {
+        SECRETARY, PRESIDENT, VOCAL
+    }
+
+    @Id @GeneratedValue
+    private long id;
+
+    @Enumerated(EnumType.ORDINAL)
+    private Role role;
+
+    @Override
+    public Long getId() {
```

Comment:
No need for getter and setters if you use the @Data annotation from Lombok. They are generated automatically. You just need to install the Lombok plugin in IntelliJ to make it aware of this.